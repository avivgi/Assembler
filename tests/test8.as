.define len=2
 LIST:   .data   6, -9, len
MAIN:           mov     r3, LIST[2]