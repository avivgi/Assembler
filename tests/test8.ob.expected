4 3
0100 00000000111000 ****!%*
0101 00000001100000 ***#%**
0102 00000110100010 **#%%*%
0103 00000000001000 *****%*
0104 00000000000110 *****#%
0105 11111111110111 !!!!!#!
0106 00000000000010 ******%
